requirements.txt

```diff
@@ -1,4 +1,4 @@
 astor
-avro
+avro==1.10.2
```

Comment:
The `avro==1.11.0` released on Oct 27, 2021 is breaking the test `test_example_message_round_trippable`, so nailing avro version to the last release (which works).